pages/machines/[id].tsx

```diff
@@ -21,24 +21,36 @@ import {
 } from '../../lib/MachineHelpers';
 import { metadata, MetadataItem } from '../../lib/metadata';
 import { useCopyToClipboard } from '../../lib/useCopyToClipboard';
+import { createMachine } from 'xstate';
+
+type ExportedMachine =
+  | StateMachine<any, any, any>
+  | (() => StateMachine<any, any, any>);
+
+const resultOf = (machine: ExportedMachine) => {
```

Comment:
Nit: maybe `getMachineFromImports` is better here?

Reply:
Yeah thats better. I was struggling for a name here haha.